lib/credstashApi.js

```diff
@@ -102,7 +104,12 @@ CredstashApi.prototype._execute = function(action, key, value) {
   } else {
     args = [action];
   }
-  return childProcess.spawnSync(COMMAND, [].concat(this.args, args, this.context));
+
+  // If we can we will use the Credstash container. If the environment variable does not exist we will attempt to use local credstash.
```

Comment:
Cool. 
We should also add tests for this kind of execution, which means that we need to run the same tests we have for this class also in this new "host mode". Talk to me if you're having trouble with that.